### PR TITLE
Fix RTL display issue on 'Create a New Theme' banner (Fixes #3509)

### LINKED ISSUE #3509: [RTL] "Create a New Theme" banner has display issues
Steps to reproduce:
1. Load the "Submit a new Theme" page https://addons-dev.allizom.org/en-US/developers/theme/submit
2. Change the language in a RTL one (i.e. https://addons-dev.allizom.org/fa/developers/theme/submit)

Expected results:
There are no display or layout issues.

Actual results:
Banner image and text are mixed up.

Notes/Issues:
Verified on FF48.0.2(Win 7 x64). Issue is reproducing all around AMO servers.
Screenshot for this issue: 
banner

### PR DESCRIPTION
**Pull request by atiqueahmedziad**
Fixes #3509

## Before
Before
## After
after

### CODE REVIEW

**Comment by atiqueahmedziad** [comment]
@tofumatt r?

File: static/css/impala/collections.less [hunk 1]
```diff
@@ -205,10 +205,12 @@ masthead.submit-theme {
...
                 -webkit-linear-gradient(#e4f5fb, #fff) repeat-x;
-    background: url(../../img/zamboni/personas/theme-promo.png) no-repeat 100% 65px,
-                linear-gradient(#e4f5fb, #fff) repeat-x;
     background-size: 50% auto, auto auto;
     p {
         width: 50%;
     }
+
+    .html-rtl & {
+    	background-position: 0% 65px, 100% 100%;
```
Comment (muffinresearch) [inline static/css/impala/collections.less]: Looks like an errant comma

Comment (muffinresearch) [inline static/css/impala/collections.less]: Ah ignore that comment - it's because there's two backgrounds.

**Comment by muffinresearch** [comment]
Test failure is unlrelated flakey test.

**Comment by muffinresearch** [comment]
@atiqueahmedziad thanks for the patch, and your patience :)

lgtm r+

**Pull request merged**